contracts/interfaces/ConsiderationEventsAndErrors.sol

```diff
@@ -137,15 +137,15 @@ interface ConsiderationEventsAndErrors {
     );
 
     /**
-     * @dev Revert with an error when insufficient ether is supplied as part of
-     *      msg.value when fulfilling orders.
+     * @dev Revert with an error when insufficient native token is supplied as
+     *      part of msg.value when fulfilling orders.
      */
-    error InsufficientEtherSupplied();
+    error InsufficientNativeTokenSupplied();
```

Comment:
I feel like this error should be `InsufficientNativeTokensSupplied` (so plural)

Reply:
Roger that, will update